pkgs/applications/virtualization/colima/default.nix

```diff
@@ -31,7 +32,7 @@ buildGoModule rec {
     '';
   };
 
-  nativeBuildInputs = [ installShellFiles makeWrapper ];
+  nativeBuildInputs = [ installShellFiles makeWrapper darwin.DarwinTools ];
```

Comment:
```suggestion
  nativeBuildInputs = [ installShellFiles makeWrapper ]
    ++ lib.optionals stdenv.isDarwin [ darwin.DarwinTools ];
```

Reply:
Yeah, I realised it just after pushing and corrected it now.